packages/orb-sync-lib/src/sync/subscriptions.ts

```diff
@@ -58,3 +58,26 @@ export async function fetchAndSyncSubscription(postgresClient: PostgresClient, o
 
   await syncSubscriptions(postgresClient, [subscription]);
 }
+
+export async function updateBillingCycle(
+  postgresClient: PostgresClient,
+  {
+    subscriptionId,
+    billingCycleStart,
+    billingCycleEnd,
+  }: {
+    subscriptionId: string;
+    billingCycleStart: string;
+    billingCycleEnd: string;
+  }
+) {
+  return postgresClient.update(
```

Comment:
I think it would be safer if this method ensures

- the billingCycleEnd date is in the future, otherwise skip
- the billingCycleStart date is in the past at the time of the check, otherwise skip
- the update statement should ideally have an additional condition in the where clause ensuring it only gets updated if the current end_date is in the past

I'd prefer having a very specific update SQL statement for this function instead of a generic upsert to ensure that